### PR TITLE
fix(css): use the dark scrollbar with `color-scheme` dark

### PR DESCRIPTION
**Pull request by malkiii**
it looks better with the dark scrollbar.

### CODE REVIEW

**Comment by leerob** [comment]
Can you show an example of what changes?

**Comment by malkiii** [comment]
I'm on windows so..
|before|after|
|-|-|
| before | after |

File: app/global.css [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 html {
   min-width: 360px;
+  color-scheme: dark;
```
Comment (leerob) [inline app/global.css:33]: Should this be in the block above for `prefers-color-scheme`? How does it work in light mode?

**Comment by malkiii** [comment]
oh! you're right, I didn't notice that there is a light/dark mode.

**Review by leerob — APPROVED** [review]

**Pull request merged**